src/features/fwiCalculator/FWICalculatorResults.tsx

```diff
@@ -6,6 +6,7 @@ import { selectPercentilesReducer } from 'app/rootReducer'
 import { PercentileMeanResultTable } from 'features/fwiCalculator/components/PercentileMeanResultTable'
 import { PercentileStationResultTable } from 'features/fwiCalculator/components/PercentileStationResultTable'
 import { ErrorMessage } from 'components/ErrorMessage'
+import { CalculationDocumentation } from './components/CalculationDocumentation'
```

Comment:
@andrea-williams we are following absolute path pattern. So please use 'features/fwiCalculator/components.......' as base path.